### PR TITLE
add put post delete document in views.py

### PR DESCRIPTION
**Pull request by QimingNie**
add put post delete document in views.py and urls.py

### CODE REVIEW

**Review by ni-sauvage — CHANGES_REQUESTED** [review]
Work looks good, nicely done! I'm just going to request that you add tests to the tests.py file as a sanity check.

**Comment by QimingNie** [comment]
> 工作看起来不错，做得很好！我只是要求您将测试添加到 tests.py 文件中作为健全性检查。

I'm trying to test it.

File: urls.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+from .views import (get_document_blocks, check_document_blocks, DocumentList,put_document_blocks,delete_document_blocks)
+
+app_name = 'api'
+
+urlpatterns = [
+    path('document/', DocumentList.as_view(), name='get_documents'),
+    path('document/', DocumentList.as_view(), name='post_document'),
+    path('document/<int:pk>', get_document_blocks, name='get_document_blocks'),
+    path('document/<int:pk>', put_document_blocks, name='put_document_blocks'),
+    path('document/<int:pk>', delete_document_blocks, name='delete_document_blocks'),
```
Comment (MaxCunningham19) [inline urls.py:11]: any chance you could change the 'document/' url to 'documents/' in line with the [REST conventions](https://restfulapi.net/resource-naming/)

**Review by MaxCunningham19 — CHANGES_REQUESTED** [review]

**Review by MaxCunningham19 — APPROVED** [review]
lgtm

**Comment by ni-sauvage** [comment]
Superseded by [#90](https://github.com/MaxCunningham19/legal_spellcheck/pull/90). Closing PR.

**Pull request closed without merging**